temba/settings.py.prod

```diff
@@ -77,6 +77,7 @@ REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.messages"] = env("V2_MESSAGES", def
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.broadcasts"] = env("V2_BROADCASTS", default="36000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.runs"] = env("V2_RUNS", default="3000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.api"] = env("V2_API", default="3000/hour")
+REST_FRAMEWORK["PAGE_SIZE"] = env.int('REST_FRAMEWORK_PAGE_SIZE', default=250)
```

Comment:
is it necessary to use `env.int`?

Reply:
by default, env variables are parsed to str. It's the same as env.bool()